depthai_examples/ros2_src/stereo_inertial_publisher.cpp

```diff
@@ -413,32 +419,59 @@ int main(int argc, char** argv) {
                                                         nnPath);
 
     std::shared_ptr<dai::Device> device;
-    std::vector<dai::DeviceInfo> availableDevices = dai::Device::getAllAvailableDevices();
-
-    std::cout << "Listing available devices..." << std::endl;
-    for(auto deviceInfo : availableDevices) {
-        std::cout << "Device Mx ID: " << deviceInfo.getMxId() << std::endl;
-        if(deviceInfo.getMxId() == mxId) {
-            if(deviceInfo.state == X_LINK_UNBOOTED || deviceInfo.state == X_LINK_BOOTLOADER) {
-                isDeviceFound = true;
-                if(poeMode) {
-                    device = std::make_shared<dai::Device>(pipeline, deviceInfo);
-                } else {
-                    device = std::make_shared<dai::Device>(pipeline, deviceInfo, usb2Mode);
-                }
-                break;
-            } else if(deviceInfo.state == X_LINK_BOOTED) {
-                throw std::runtime_error("\" DepthAI Device with MxId  \"" + mxId + "\" is already booted on different process.  \"");
+    if(useWithIP) // Connecting to a camera with specific IP, requires static IP config beforehand
+    {
+        auto deviceInfo = dai::DeviceInfo(ipAddress);
```

Comment:
What will happen if `useWithIp` is set but ipAddress is not set ? 
Same for `useWithMxID`

Reply:
Good condition to check. will test and update it.